src/components/demo-iframe/demo-iframe.js

```diff
@@ -26,35 +22,53 @@ export default class DemoIframe extends React.Component {
   }
 
   render() {
-    const { props } = this;
+    let { src, title, gl } = this.props;
+    const { mapboxAccessToken } = this.state;
     // check to see if the src is making a request to mapbox api
-    const makeRequest = props.src.indexOf('MapboxAccessToken') > -1;
+    const makeRequest = src.indexOf('MapboxAccessToken') > -1;
     // if it's making a request and the mapboxAccessToken hasn't loaded yet
     // return null (nothing) until it has loaded
-    if (makeRequest && !this.state.mapboxAccessToken) {
+    if (makeRequest && !mapboxAccessToken) {
       return null;
     }
     // if the src has an `access_token` value, replace it with mapboxAccessToken
     // otherwise use the src as is
-    const src = makeRequest
-      ? props.src.replace('MapboxAccessToken', this.state.mapboxAccessToken)
-      : props.src;
+    src = makeRequest
+      ? src.replace('MapboxAccessToken', mapboxAccessToken)
+      : src;
+
+    const iframeHeight = 400;
```

Comment:
Should this be a default prop value instead? Otherwise defining the height like in the [last test case](https://github.com/mapbox/dr-ui/pull/395/files#diff-3ccba7f92bbe8bb3c98f3082aa91935efade014657d4e9860f50dd8127ce7dcdR38 ) won't work, right? Or I guess the alternative would be to remove `height` from the test case and always use a height of 400. I don't have a strong opinion either way!

Reply:
@colleenmcginnis Oops, didn't mean to add it to that testcase! I'll remove. We had DemoIframes to always be 400px and I haven't heard a reason to change that. Unless you think we should make it variable?

Reply:
Nope! Let's leave it at 400px. I do love consistency ... including in the heights of my iframes!